diff --git a/completions/man b/completions/man
--- a/completions/man
+++ b/completions/man
@@ -53,9 +53,7 @@ _man()
         return
     fi
 
-    local manpath="$MANPATH"
-    [[ -z $manpath ]] && \
-        manpath=$( manpath 2>/dev/null || command man -w 2>/dev/null )
+    local manpath=$( manpath 2>/dev/null || command man -w 2>/dev/null )
     [[ -z $manpath ]] && manpath="/usr/share/man:/usr/local/share/man"
 
     # determine manual section to search
